### PR TITLE
Fix 3rd party licenses updater (action)

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
Opened in favour of https://github.com/erlef/setup-beam/pull/201 (since I wanna use `workflow_dispatch`).

### CODE REVIEW

File: .github/workflows/update_3rd_party_licenses.yml [hunk 1]
```diff
@@ -17,3 +17,5 @@ jobs:
           node-version: '14'
       - run:
           ./.github/workflows/update_3rd_party_licenses.sh
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (paulo-ferraz-oliveira) [inline .github/workflows/update_3rd_party_licenses.yml:21]: The workflow only started failing recently, due to not having this... 🤷

**Review by starbelly — APPROVED** [review]

**Comment by paulo-ferraz-oliveira** [comment]
Thus we shall soon have a new (automated) pull request, if everything goes according to plan 😄  Thanks, @starbelly.

**Comment by paulo-ferraz-oliveira** [comment]
Still not everything working as it should: https://github.com/erlef/setup-beam/actions/runs/4936735135/jobs/8824633938 😢

**Comment by paulo-ferraz-oliveira** [comment]
Released in `v1.16.0` / `v1.16`. Tag `v1` moved up...

**Pull request merged**